### PR TITLE
NATIVE-386: Make slides legally legal

### PR DESCRIPTION
**Pull request by klinzo**
Suggestions for improvement are appreciated. I'd move the snackbar to a separate task.

### CODE REVIEW

File: src/routes/intro/IntroSettings.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+import * as React from 'react'
+import type { ThemeType } from '../../modules/theme/constants/theme'
+import type TFunction from 'react-i18next'
+import styled, { type StyledComponent } from 'styled-components/native'
+import openPrivacyPolicy from '../settings/openPrivacyPolicy'
+
+const Padding: StyledComponent<{}, ThemeType, *> = styled.View`
+  flex: 1;
+  padding: 10px;
+  flex-direction: column;
```
Comment (Schedulaar) [inline src/routes/intro/IntroSettings.js]: On my smaller screen of the emulator, the content is not aligned perfectly:

image

You can use `justify-content: space-between;` here and make ItemContainer and LinkContainer normal Views (without any extra styling), then I get this result (together with changing the padding below):

image


```suggestion
  flex-direction: column;
  justify-content: space-between;
```

File: src/routes/intro/IntroSettings.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+`
+
+const LinkContainer = styled.View`
+  flex: 1;
+  justify-content: center;
+`
+
+const Link = styled.Text`
+  padding-top: 20px;
+  color: blue;
```
[the remark this answers is not in the document]
Comment (klinzo) [reply src/routes/intro/IntroSettings.js]: Sounds like a good proposal, will try it out.

File: src/routes/intro/IntroSettings.js (hunk 1 above)
Comment (klinzo) [reply src/routes/intro/IntroSettings.js]: Good suggestions, looks nice :)

**Comment by klinzo** [comment]
@Schedulaar should be fixed now :)

File: src/modules/app/components/Navigator.js [hunk 3]
```diff
@@ -30,30 +30,31 @@ class Navigator extends React.Component<PropsType, StateType> {
...
+
     if (!language) {
       throw Error('The contentLanguage has not been set correctly by I18nProvider!')
     }
 
     if (!introShown) {
       this.appNavigationContainer = createAppNavigationContainer({ initialRouteName: 'Intro' })
     } else {
-      if (settings.errorTracking) {
+      if (settings.allowSentry) {
```
Comment (maxammann) [inline src/modules/app/components/Navigator.js]: I think the name errorTracking is better if we decide to switch away from Sentry for some reason in the future

File: src/modules/i18n/components/I18nProvider.js [hunk 4]
```diff
@@ -102,14 +104,15 @@ class I18nProvider extends React.Component<PropsType, StateType> {
...
+    const { errorMessage, contentLanguageInitialized } = this.state
+    if (errorMessage) {
       return <Text>{this.state.errorMessage}</Text>
     }
 
     return (
       <I18nextProvider i18n={this.i18n}>
         <MomentContext.Provider value={this.momentFormatter}>
-          {this.props.children}
+          {contentLanguageInitialized ? this.props.children : null}
```
Comment (maxammann) [inline src/modules/i18n/components/I18nProvider.js]: :+1:

File: src/routes/intro/IntroContainer.js [hunk 5]
```diff
@@ -67,89 +93,88 @@ class Intro extends React.Component<PropsType, StateType> {
...
+
     try {
-      if (errorTracking) {
+      if (allowSentry) {
         const sentry = new SentryIntegration()
         await sentry.install()
       }
 
-      if (useLocationAccess) {
+      if (proposeNearbyCities) {
```
Comment (maxammann) [inline src/routes/intro/IntroContainer.js]: :+1:

File: src/routes/intro/SlideFooter.js [hunk 6]
```diff
@@ -105,18 +93,55 @@ class SlideFooter extends React.Component<PropsType> {
     </DotsContainer>
   }
 
-  render () {
-    const { currentSlide, slideCount, theme } = this.props
+  renderStandardFooter = (): React.Node => {
```
Comment (maxammann) [inline src/routes/intro/SlideFooter.js]: Can be extracted into separate components like: 'StandartFooter', 'CustomizableSettingsFooter'

Comment (klinzo) [reply src/routes/intro/SlideFooter.js]: imo it's fine like this

Comment (klinzo) [reply src/routes/intro/SlideFooter.js]: Imo its fine like it is

**Pull request merged**